composer.json

```diff
@@ -160,7 +160,7 @@
         "symfony/yaml": "^6.3",
         "terminal42/escargot": "^1.4.1",
         "terminal42/service-annotation-bundle": "^1.1",
-        "toflar/psr6-symfony-http-cache-store": "^3.0 || ^4.0",
+        "toflar/psr6-symfony-http-cache-store": "^4.0",
```

Comment:
Version 3 is not compatible with Symfony 6, therefore it is not installable anymore.